src/app-common/components/common/search/search.reducer.js

```diff
@@ -0,0 +1,30 @@
+const initialState = {
+    searchValue: '',
+    countMovies: null,
+    selectedSearchType: 'title'
+};
+
+const search = (state = initialState, action) => {
+    switch (action.type) {
+        case 'SEARCH_MOVIES_BY_TYPE':
+            return {
+                ...state,
+                selectedSearchType: action.payload
+            }
+        case 'GET_SEARCH_VALUE':
+            return {
+                ...state,
+                searchValue: action.payload
+            }
+        case 'SEARCH_MOVIES':
+            return {
+                ...state,
+                movies: action.payload,
```

Comment:
We don't need to set movies here, because we don't use _movies_ from search.reducer.js in any our component.